source/state_representation/README.md

```diff
@@ -197,9 +207,9 @@ StateRepresentation::JointState jsdiff = js1 - js2;
 StateRepresentation::JointState jsscaled = lambda * js1;
 ```
 
-Multiplication, that does not have a physical meaning, is not implemented.
+Multiplication of joint states doesn't have a physical meaning and is, therefore, not implemented.
 
-### Conversion between state variables
+### Conversion between Joint state variables
```

Comment:
(see earlier comment, I think "... between joint state variables" is more appropriate in this context)